soccer/gameplay/situational_play_selection.py

```diff
@@ -247,7 +247,7 @@ def closestReciever(self):
     def possesses_the_ball(self,
                            ballPos,
                            robot,
-                           distThresh=0.14,
+                           distThresh=0.2,
```

Comment:
There was some flip flopping in and out of possession due to our poor capture performance so I made the thresholds a bit more forgiving, changing the preempt delay is also an option.